cmd/s2i/s2i.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	"github.com/openshift/source-to-image/pkg/cmd/cli"
+)
+
+func main() {
+	rand.Seed(time.Now().UTC().UnixNano())
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
```

Comment:
@jim-minter indeed since Go 1.5 ([release notes](https://golang.org/doc/go1.5)):
> By default, Go programs run with GOMAXPROCS set to the number of cores available; in prior releases it defaulted to 1.

This effectively does nothing, and, as Ben suggests, it must come as inheritance from the early v3 days ;)